src/test/java/com/work/management/service/employee/impl/EmployeeServiceImplTest.java

```diff
@@ -48,10 +53,25 @@ public void updateEmployeeEntity_forExistingEmployee_returnsUpdatedEmployee() {
     assertThat(actualEmployeeDto).isEqualTo(employeeDto);
   }
 
-  private static EmployeeDto getEmployeeDto(Employee employee) {
-    EmployeeDto employeeDto = new EmployeeDto();
-    BeanUtils.copyProperties(employee, employeeDto);
-    return employeeDto;
+  @Test
+  public void bulkUpdate_forExistingEmployees_returnUpdatedEmployeesList() {
```

Comment:
Add a test case when one of the employees is not found.
Also what will happen if same employee is present more than once in the list for updating employees?
Add test case and code for this scenario.